js/bootstrap-datepicker.js

```diff
@@ -141,13 +141,12 @@
 			this.picker.addClass('datepicker-rtl');
 		}
 
-		this.viewMode = this.o.startView;
-
-		if (this.o.calendarWeeks)
-			this.picker.find('thead .datepicker-title, tfoot .today, tfoot .clear')
-						.attr('colspan', function(i, val){
-							return parseInt(val) + 1;
-						});
+		if (this.o.calendarWeeks) {
+			this.picker.find('.datepicker-days .datepicker-switch, thead .datepicker-title, tfoot .today, tfoot .clear')
```

Comment:
I moved this. See below: https://github.com/eternicode/bootstrap-datepicker/pull/1893/commits/4c70acaad66f3bdbbdd91d0c70b44d73c535c223#diff-dd513a8bab7ad1033c8784c4a1b9ce15L828